pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -55,6 +55,11 @@ func CmdCreate() *cobra.Command {
 	cmd.Flags().StringSlice("allowed_buckets", nil,
 		"Set the user allowed buckets list (use commas or multiple flags)")
 	cmd.Flags().String("default_resource", "", "Set the default resource, on which new buckets will be created")
+	cmd.Flags().Bool("nsfs_account_config", false, "This flag is for creating nsfs account")
+	cmd.Flags().Int("uid", -1, "Set the nsfs uid")
+	cmd.Flags().Int("gid", -1, "Set the nsfs gid")
+	cmd.Flags().String("new_buckets_path", "/", "Change the default nsfs mount path")
+	cmd.Flags().Bool("nsfs_only", true, "Set if this account for nsfs_only")
```

Comment:
sound weird... maybe Set if this account is used only for nsfs. maybe @romayalon can help with all of the above descriptions.